### DISCUSSION TITLE
Selection and ColorScheme

### DISCUSSION THREAD

**Discussion by thomasproductions** [tui-cs/Terminal.Gui#1688] [Q&A] [answered]
Hey all,

When I add a list AND change the color scheme on it, the Selection marker is always black for foreground and background, is there a way to change this?
or am I missing somthing?
thanks!

            
            iList.Add("test 1");
            iList.Add("test 2");
            var color2 = new ColorScheme { Normal = Application.Driver.MakeAttribute(Color.White, Color.Blue) };

            text = new ListView(iList) { X = 2, Y = 1, Width = Dim.Fill()-1, Height = Dim.Fill() - 1 };
            text.ColorScheme = color2;
            
           window.Add(text);

![image](https://user-images.githubusercontent.com/61852391/166123172-38c899f3-2809-48b4-aa65-d19c14f654ed.png)

**Comment by BDisp** [comment, marked as answer]
Or you use the default `ColorScheme` for the `ListView` or you also must provide the other colors for it, `Focus`, `HotNormal`, `HotFocus`, etc.

**Reply by thomasproductions** [reply]
thanks! for some reason I equated focus to listview focus and not selection focus. this helped!
